parsers/php.c

```diff
@@ -1034,8 +1034,9 @@ static boolean parseClassOrIface (tokenInfo *const token, const phpKind kind,
 	}
 
 	inheritance = vStringNew ();
-	/* skip until the open bracket and assume every identifier (not keyword)
-	 * is an inheritance (like in "class Foo extends Bar implements iA, iB") */
+	/* Accept only identifiers, keywords, and/or commas; and capture the identifiers.
+	 * This aproach is more robust than "Capture the identifier until an open bracket
+	 * is found". */
```

Comment:
the second sentence isn't really important and IMO only reflects what the old (removed) code did, so it shouldn't be there.

I'd myself just alter it like that: 
> read every identifiers, keywords and commas, and assume each identifier (not keyword) is an inheritance (like in "class Foo extends Bar implements iA, iB")